docs/conf.py

```diff
@@ -0,0 +1,70 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+import check_shapes
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'check_shapes'
+copyright = '2022, The GPflow Contributors'
+author = 'The GPflow Contributors'
+
+# The full version, including alpha/beta/rc tags
+release = check_shapes.__version__
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    'autoapi.extension'
+]
+
+autoapi_dirs = ['../check_shapes']
+autoapi_add_toctree_entry = False
+autodoc_typehints = 'description'
+autoapi_options = [
+    "members",
+    "private-members",
+    "special-members",
+    "imported-members",
+    "show-inheritance",
+]
+autoapi_python_class_content = "both"
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'alabaster'
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
```

Comment:
FYI I managed to get a version navbar for [trieste](https://secondmind-labs.github.io/trieste/0.12.0/index.html) without _too_ much pain. Also, a github link.

Reply:
> FYI I managed to get a version navbar for [trieste](https://secondmind-labs.github.io/trieste/0.12.0/index.html) without _too_ much pain. Also, a github link.

Yeah, GPflow has the same navbar version switcher. However, it is currently broken. Apparently the PyData people aren't very good at backwards compatibility and right now I don't have the energy to deal with that. Maybe another day.
https://github.com/pydata/pydata-sphinx-theme/issues/932